screeps-game-api/src/constants.rs

```diff
@@ -250,6 +250,9 @@ pub enum Terrain {
     Plain = 0,
     Wall = 1,
     Swamp = 2,
+    SwampWall = 3,
+    Lava = 4,
```

Comment:
I don't *think* this should be necessary? I'm against including if if it doesn't occur just because it forces any consumers of `Terrain` to insert an extra case `Terrain::Lava => unreachable!(),` whenever matching on it which will never be triggered.

The screeps team has mentioned on slack that `TERRAIN_MASK_LAVA` has never been used, and that there are no current plans for using it. It's also missing from [Room.Terrain.get](https://docs.screeps.com/api/#Room.Terrain.get)'s listed output.

Reply:
It's a question of whether you prefer to implement 1:1 the constants or clean-up the API on our end. I'm all for making the usage easier rather than going for 1:1 translation.

Reply:
In this case I would go for "cleanup", but I _think_ 1:1 would agree here too.  We're only ever returning the terrain, never passing it into the game, and the only place it's returned is documented to not return 4. Even though the constant exists, it's documented to never be used?